packages/mdc-list/index.ts

```diff
@@ -135,7 +135,7 @@ class MDCList extends MDCComponent<MDCListFoundation> {
         }
       },
       focusItemAtIndex: (index) => {
-        const element = this.listElements[index] as HTMLElement;
+        const element = this.listElements[index] as HTMLElement | null;
```

Comment:
You are technically correct! 😀  Fixed.

There's a subtle but important difference between a type assertion with and without `| null` (or `| undefined`):

```ts
// This tells the TypeScript compiler that `element` will
// NEVER be null or undefined, so it won't yell at you
// if you try to access a property on `element`, which is dangerous.
const element = this.listElements[index] as HTMLElement;

// This tells tsc that `element` might not have a value, so it should
// yell at you if you try to access a property without a truthy check.
const element = this.listElements[index] as HTMLElement | undefined;
```

In this specific case, it doesn't matter quite as much because we already have a truthy check below (`if (element) { ... }`), but if we ever refactor this code, we could easily forget that `element` might be `undefined`.

Personally, I'd rather be semantically accurate by declaring that the type might be `undefined`.